js-chat-app.yml

```
@@ -0,0 +1,12 @@
+---
+name: JS Chat Application
+desc: Basic chatroom application developed using MEAN Stack
+site: https://github.com/tanmayvij/mean-chat-app
+tags:
+- javascript
+- chat
+- mongodb
+- nodejs
+upforgrabs:
+  name: up-for-grabs
```

@tanmayvij This should be `upforgrabs` as per your label link.

```suggestion
  name: upforgrabs
```